selfdrive/car/hyundai/interface.py

```diff
@@ -247,6 +247,11 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[], experi
       ret.steerRatio = 16.
       tire_stiffness_factor = 0.65
       CarInterfaceBase.configure_torque_tune(candidate, ret.lateralTuning)
+    elif candidate == CAR.KIA_SPORTAGE_HYBRID_5TH_GEN:
+      ret.mass = 1767. + STD_CARGO_KG  # SX Prestige trim support only
```

Comment:
same here, shouldn't all be supported?

Reply:
Yes, only SX Prestige comes with SCC. All lower trims of the Sportage Hybrid only have regular Cruise Control.

Reply:
https://www.kiamedia.com/us/en/models/sportage-hev/2023/features
![image](https://user-images.githubusercontent.com/47793918/196063366-595942a1-eca8-49a4-b369-1f103e7cae74.png)